src/wasm-interpreter.h

```diff
@@ -2228,6 +2228,186 @@ class ModuleInstance
     : ModuleInstanceBase(wasm, externalInterface) {}
 };
 
+static const Name NONSTANDALONE_FLOW("Binaryen|nonstandalone");
```

Comment:
this const and the typedef after it are now in the general global scope, while before they were just in Precompute's global scope. Please move them into the new class.

Reply:
I might be lacking C++ experience here, but iiuc moving the name to the class has the side-effect that it can't be static anymore, since it isn't an integer, so I ended up with one `const` definition inside the class and another `static const` definition of the same name in the precompute pass since it cannot reference the former, which has the potential to break if the respective string is changed at either place. Is there a better way?

Reply:
You can have a static member inside the class, but you have to initialize it outside the class with `type ClassName::member = initial_value`. Probably best to have a new .cpp file that holds this and the rest of the class implementation.

Reply:
Thanks, I think I figured it out. Always a pleasure to learn some C++ from you guys, enjoying it :)